docs/src/main/paradox/jms.md

```diff
@@ -40,256 +40,236 @@ Gradle
 
 ## Usage
 
-The JMS message model supports several types of message body (see @javadoc[javax.jms.Message](javax.jms.Message)) and Alpakka currently supports messages with a body containing a @javadoc[String](java.lang.String) object.
+The JMS message model supports several types of message body (see @extref(javaee-api:javax.jms.Message)) and Alpakka currently supports messages with a body containing a @extref[String](java-api:java.lang.String), 
+a @extref[Serializable](java-api:java.io.Serializable) object, a @extref[Map](java-api:java.util.Map) or a byte array.
 
-Use the case class @scaladoc[JmsTextMessage](akka.stream.alpakka.jms.JmsTextMessage$) to wrap the messages you want to send and optionally set their properties (see below for an example).
+Use a case class with the subtype of @scaladoc[JmsMessage](akka.stream.alpakka.jms.JmsMessage$) to wrap the messages you want to send and optionally set their properties (see below for an example).
 
 ### Sending messages to a JMS provider
 
-First define a jms @javadoc[javax.jms.ConnectionFactory](javax.jms.ConnectionFactory) depending on the implementation you're using. Here we're using Active MQ.
+First define a jms `ConnectionFactory` depending on the implementation you're using. Here we're using Active MQ.
 
 Scala
-: @@snip ($alpakka$/jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsConnectorsSpec.scala) { #connection-factory }
+: @@snip (../../../../jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsConnectorsSpec.scala) { #connection-factory }
 
 Java
-: @@snip ($alpakka$/jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java) { #connection-factory }
+: @@snip (../../../../jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java) { #connection-factory }
 
 
-Create a sink, that accepts and forwards @javadoc[String](java.lang.String)s to the JMS provider.
+Create a sink, that accepts and forwards @extref[String](java-api:java.lang.String)s to the JMS provider.
```

Comment:
Please use the newly configured `@javadoc[String](java.lang.String)` style links.

Reply:
Fixed